client/src/main/java/com/alibaba/nacos/client/naming/utils/InitUtils.java

```diff
@@ -88,7 +90,7 @@ public String call() {
         tmpNamespace = TemplateUtils.stringEmptyAndThenExecute(tmpNamespace, new Callable<String>() {
             @Override
             public String call() {
-                return UtilAndComs.DEFAULT_NAMESPACE_ID;
+                return DEFAULT_NAMESPACE_ID;
```

Comment:
default namespace id may be reuse.
Use old one maybe better.

Reply:
I'm going to put variables like these into common module.